File: clients/java/examples/collections-list-collection-names.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import java.util.List;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // List collections
    List<String> collectionNames = database.listCollectionNames();
  }
}

File: clients/go/examples/collections-list-collection-names.go
package main

import (
	"context"
	"fmt"
	"log"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	// Get a database
	client := astra.NewClient()

	database := client.Database(
		"**API_ENDPOINT**",
		options.API().SetToken("**APPLICATION_TOKEN**"),
	)

	// List collections
	names, err := database.ListCollectionNames(ctx)
	if err != nil {
		log.Fatal(err)
	}

	fmt.Println(names)
}
